sass/custom/_terminology_tooltip.scss

```diff
@@ -0,0 +1,46 @@
+.terminology {
+  position: relative;
+  border-bottom: 2px dotted #049cdb;
+  cursor: help;
+
+  &:hover .terminology-tooltip {
+    visibility: visible;
+    opacity: 1;
+  }
```

Comment:
`hover` is not supported on touch devices. We need some javascript to toggle a class instead of using `hover`.

Reply:
It works perfectly on phones by indicating you can click it which will open it.

Reply:
On my iPhone, I need to hold the word to display the tooltip and I need to focus another link or button to close it.

Reply:
On my iPhone it just works.

Reply:
I cannot reproduce it

https://user-images.githubusercontent.com/195327/224477429-33b66540-4361-4d5f-b078-8a5d37139fbb.MOV

Reply:
I've now completed multiple tests of the preview URL on multiple browsers and mobile devices. Including iOS & Android mobile phones and tablets. In all cases, they worked as expected.

For that reason, I'll continue on merging this one in.